model/permission.go

```diff
@@ -645,4 +647,12 @@ func initializePermissions() {
 
 func init() {
 	initializePermissions()
+	ModeratedPermissions = map[string]bool{
```

Comment:
Will be switching this to https://github.com/mattermost/mattermost-server/pull/13845/files#diff-8c79414d09c4d39a11d406480ac289e0R655.